test/doppelganger-test.js

```diff
@@ -1,14 +1,15 @@
 const { expect } = require("chai");
 const { ethers } = require("hardhat");
+const { smock } = require('@defi-wonderland/smock');
 
 describe("Doppelganger", function () {
-  let mockContract, doppelgangerContract;
+  let myFake;
+  let doppelgangerContract;
 
   beforeEach(async function () {
-    // Original NFT contract
-    const ERC721Mock = await ethers.getContractFactory("ERC721Mock");
-    mockContract = await ERC721Mock.deploy();
-    await mockContract.deployed();
+    // Original NFT contract (mocked)
+    const myErc721Factory = await ethers.getContractFactory("ERC721");
+    myFake = await smock.fake(myErc721Factory);
```

Comment:
I think we can include the Mock factory like this in the docs and remove the `getContractFactory` call

> import { FakeContract, smock } from '@defi-wonderland/smock';

Reply:
change is here: 
https://github.com/johnmosesman/nftrusty/pull/2

I couldn't get the `FakeContract` fixture to work as I wasn't sure what to bind to the `MyContract` template argument.  

I tried importing ERC721.json from the "artifacts" folder but that yielded an inscrutable run-time error

I was still able to reduce this to a single line with this style: 
`const myFake = await smock.fake('MyContract');`